.editorconfig

```diff
@@ -50,17 +53,19 @@ align_array_table = false
 ## Other indentation settings
 never_indent_before_if_condition = false
 never_indent_comment_on_if_branch = false
+keep_indents_on_empty_lines = false
+allow_non_indented_comments = true
```

Comment:
What's the usecase for non-indented comments?

Reply:
No particular use case in mind except I see no reason why someone shouldn't be allowed to do it if they want to.

This stuff is all forced on anyone who has auto-formatting enabled, so I think we should only turn a setting on if we want to force all code to conform to it.